src/pybind/mgr/mgr_module.py

```diff
@@ -465,7 +400,92 @@ def register(self, instance=False):
         )
 
 
-class MgrStandbyModule(ceph_module.BaseMgrStandbyModule):
+class CPlusPlusHandler(logging.Handler):
+    def __init__(self, module_inst):
+        super(CPlusPlusHandler, self).__init__()
+        self._module = module_inst
+        self.setFormatter(logging.Formatter("[{}] [%(levelname)-4s] [%(name)s] %(message)s"
```

Comment:
Wouldn't `%(module)s` work? I was thinking on something like `[dashboard.controller.home]`?

Reply:
`$(module)s` only shows the last part of the module path. For `dashboard.controller.home` it only shows `home`